python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -353,3 +353,95 @@ def _doConvolve(exposure, kernel):
         afwMath.convolve(outExp.getMaskedImage(), exposure.getMaskedImage(), kern, convCntrl)
 
         return outExp, kern
+
+
+class DecorrelateALKernelMapperTask(DecorrelateALKernelTask):
+    """Task to be used as an ImageMapperSubtask for computing
+    A&L decorrelation on subimages on a grid across a A&L difference image.
+
+    This task subclasses DecorrelateALKernelTask in order to implement
+    all of that task's configuration parameters, as well as its `run` method.
+    """
+    ConfigClass = DecorrelateALKernelConfig
```

Comment:
No, that config is passed to the ImageMapReduceClass to set its mapperSubtask to this class.

Confusing, right?